src/libs/vpatterndb/vpiece.h

```diff
@@ -72,102 +72,109 @@ class VPointF;
+                                                const VContainer *data, VSAPoint &point) const;
+    bool                     getNotchPreviousSAPoints(const QVector<VPieceNode> &path, int index,
+                                                         const VSAPoint &notchSAPoint,
                                      const VContainer *data, VSAPoint &point) const;
-    int GetPassmarkNextSAPoints(const QVector<VPieceNode> &path, int index, const VSAPoint &passmarkSAPoint,
-                                const VContainer *data, VSAPoint &point) const;
-    bool GetSeamPassmarkSAPoint(const VSAPoint &previousSAPoint, const VSAPoint &passmarkSAPoint,
-                                const VSAPoint &nextSAPoint, const VContainer *data, QPointF &point) const;
-
-    bool IsPassmarkVisible(const QVector<VPieceNode> &path, int passmarkIndex) const;
-
-    QVector<QLineF> CreatePassmark(const QVector<VPieceNode> &path, int previousIndex, int passmarkIndex, int nextIndex,
-                                   const VContainer *data,
-                                   const QVector<QPointF> &seamAllowance = QVector<QPointF>()) const;
-    QVector<QLineF> SAPassmark(const QVector<VPieceNode> &path, VSAPoint &previousSAPoint,
-                               const VSAPoint &passmarkSAPoint, VSAPoint &nextSAPoint, const VContainer *data,
-                               int passmarkIndex, const QVector<QPointF> &seamAllowance = QVector<QPointF>()) const;
-    QVector<QLineF> BuiltInSAPassmark(const QVector<VPieceNode> &path, const VSAPoint &previousSAPoint,
-                                      const VSAPoint &passmarkSAPoint, const VSAPoint &nextSAPoint,
-                                      const VContainer *data, int passmarkIndex) const;
-
-    static int IsCSAStart(const QVector<CustomSARecord> &records, quint32 id);
+    int                      getNotchNextSAPoints(const QVector<VPieceNode> &path, int index,
+                                                     const VSAPoint &notchSAPoint,
+                                                     const VContainer *data, VSAPoint &point) const;
+    bool                     getSeamNotchSAPoint(const VSAPoint &previousSAPoint, const VSAPoint &notchSAPoint,
+                                                    const VSAPoint &nextSAPoint,
+                                                    const VContainer *data, QPointF &point) const;
+
+    bool                     isNotchVisible(const QVector<VPieceNode> &path, int notchIndex) const;
+
+    QVector<QLineF>          createNotch(const QVector<VPieceNode> &path, int previousIndex, int notchIndex,
+                                            int nextIndex, const VContainer *data,
+                                            const QVector<QPointF> &seamAllowance = QVector<QPointF>()) const;
+    QVector<QLineF>          createSaNotch(const QVector<VPieceNode> &path, VSAPoint &previousSAPoint,
```

Comment:
createSANotch

Reply:
Logically you would think it should be SA, but abbreviations should follow:

No All Upper Case Abbreviations
When confronted with a situation where you could use an all upper case abbreviation instead use an initial upper case letter followed by all lower case letters. No matter what.
Justification
People seem to have very different intuitions when making names containing abbreviations. It's best to settle on one strategy so the names are absolutely predictable.
Take for example NetworkABCKey. Notice how the C from ABC and K from key are confused. Some people don't mind this and others just hate it so you'll find different policies in different code so you never know what to call something.

Example
   class FluidOz             // NOT FluidOZ
   class NetworkAbcKey       // NOT NetworkABCKey

Reply:
ok